coalahtml/coala_html.py

```diff
@@ -35,7 +35,7 @@ def main():
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:
+    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
```

Comment:
Yes, sure.
>Anyway this is going to be in 58f9d34

As I see, there are two alternatives, one is that you only check that the file structure is the same. Then, maybe, it'll be easier for people to hack on stuff, if they wanted to. Another approach can be to check the file content as well. This way, if you'll be changing files in the new version you'll update them, but it's more costly and will give no hacking ability.